cmake/toolchain/llvm/target.cmake

```
@@ -42,3 +42,12 @@ if(DEFINED triple)
 
   unset(triple)
 endif()
+
+if(CONFIG_LLVM_LIBGCC)
+  set(RUNTIME_LIB "libgcc")
```

minor nit, please use lower case naming for variables intended to only be used in this file.
```suggestion
  set(runtime_lib"libgcc")
```

---

Fixed.